src/categories.toml

```diff
@@ -297,6 +297,33 @@ formats. Format-specific parsers belong in other, more specific \
 categories.\
 """
 
+[rendering]
+name = "Rendering"
+description = """
+Crates related to rendering 2D or 3D graphics, \
+usually on the screen and with the help of the graphics card.\
```

Comment:
There are two domains when it comes to rendering: "real-time" rendering (as in video games) and "offline" rendering, for example when you render a movie like Toy Story into a movie file.

The category "images" would be more appropriate for example to resize or change the colors of an image, or for face regonition, etc.

Nobody familiar with offline rendering would have the idea to look in a category named "image", because it has nothing to do with that.